.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
Depending on how often you feel like dealing with these, I suggest lowering this to weekly or every few days 🙂

Reply:
I'll leave it daily, and if it gets annoying, I'll change it 👍